cognite/client/data_classes/extractionpipelineruns.py

```diff
@@ -0,0 +1,84 @@
+from cognite.client.data_classes._base import *
+from cognite.client.data_classes.shared import TimestampRange
+
+
+class ExtractionPipelineRun(CogniteResource):
+    """A representation of an extraction pipeline run.
+
+    Args:
+        external_id (str): The external ID of the extraction pipeline.
```

Comment:
why do we allow only `external_id` for pipeline run but support `id` for pipeline itself?

Reply:
The backend only accepts external ids. Since it's required in create there is no need for internal id as a fallback, external ids can always be assumed to be there.